internal/integration/hclsqlspec/migrate_test.go

```diff
@@ -18,6 +18,21 @@ modify_table {
 		}
 	}
 }
+
+add_table {
+	table "products" {
+		column "sku" {
+			type = "string"
+		}
+		index "sku" {
+			unique = true
+		}
+	}
+}
+
+drop_table {
+	table = "products"
+}
```

Comment:
I wonder, if something like the following looks better:

```suggestion
drop_table "products" {}

drop_table "products" {
	// attrs.
}

modify_table "products" {
	add_column "..." {
		...
	}
}
```

Reply:
for drop/add table I agree, changing.

for add_column, i think its nice that the user basically embeds a `*sqlspec.Column` and we can reuse that component and syntax without much magic.

Reply:
Looking at an option to support 

```go
type AddTable struct {
  *sqlspec.Table
}
```

currently `Resource.As` breaks on this, but I'm investigating why.

Reply:
> for add_column, i think its nice that the user basically embeds a *sqlspec.Column and we can reuse that component and syntax without much magic.

Who's the user here? Also, what about `drop` and `modify`? Do you think they need to have a different structure?

```hcl
modify_table "t" {
    add_column "c1" {
        type = "int"
        ...
    }

    drop_column "c2" {}

    modify_column "c3" {
        ...
    }
}
```